### PR TITLE
Created, registered and migrated necessary models

### PR DESCRIPTION
**Pull request by IsmailSadiqO**
I created the following models:
UserProfile in AuthenticationSystem/models.py
Capsule, CapsuleMedia, Subscription, Comment in TimeCapsuleManagement/models.py

### CODE REVIEW

File: AuthenticationSystem/models.py [hunk 1]
```diff
@@ -1,3 +1,11 @@
 from django.db import models
+from django.contrib.auth.models import User
 
 # Create your models here.
+
+
+class UserProfile(User):
```
Comment (Harishcruise) [inline AuthenticationSystem/models.py:7]: So the user class will contains all the other fields right? like fullname, email, etc?
and do we really need phone number?
If possible can u keep the user profile class empty with only uuid or id field, so the authentication team will define the class.

File: capsule_content/2024/02/25/20230501_220748_2.png
Comment (Harishcruise) [inline capsule_content/2024/02/25/20230501_220748_2.png:1]: Where are you using this Image?

File: TimeCapsuleManagement/models.py [hunk 2]
```diff
@@ -1,3 +1,57 @@
...
+
+
+class CapsuleContent(models.Model):
+    FILE_TYPES = (
+        ('photo', 'Photo'),
+        ('video', 'Video'),
+        ('text', 'Text'),
+    )
+    file_type = models.CharField(max_length=20, choices=FILE_TYPES)
+    file = models.FileField(upload_to='capsule_content/%Y/%m/%d/')
```
Comment (Harishcruise) [inline TimeCapsuleManagement/models.py:32]: Keep this field as a public url, that should work.

**Review by Harishcruise — CHANGES_REQUESTED** [review]

File: AuthenticationSystem/models.py (hunk 1 above)
Comment (IsmailSadiqO) [reply AuthenticationSystem/models.py:7]: Sure

File: capsule_content/2024/02/25/20230501_220748_2.png
Comment (IsmailSadiqO) [reply capsule_content/2024/02/25/20230501_220748_2.png:1]: It was just a test I uploaded an image and its stored in a folder. When we set up s3 the images will be uploaded there

File: TimeCapsuleManagement/models.py (hunk 2 above)
Comment (IsmailSadiqO) [reply TimeCapsuleManagement/models.py:32]: Its set up to store the image on your local device. When we set up s3 we can change the path

**Review by Harishcruise — APPROVED** [review]

**Pull request merged**